src/app-album-comments.component.js

```diff
@@ -0,0 +1,36 @@
+import template from './app-album-comments.html'
+
+export const AppAlbumCommentsComponent = {
+  controller : function ($state, SpotifyService) {
```

Comment:
The controller should be a class, also should be marked with `ngInject` to have the proper dependency injection